core/src/main/java/org/geysermc/geyser/translator/protocol/bedrock/entity/player/BedrockActionTranslator.java

```diff
@@ -75,6 +75,10 @@ public void translate(GeyserSession session, PlayerActionPacket packet) {
                 attributesPacket.setRuntimeEntityId(entity.getGeyserId());
                 attributesPacket.getAttributes().addAll(entity.getAttributes().values());
                 session.sendUpstreamPacket(attributesPacket);
+
+                // Bounding box must be sent after a player dies and respawns since 1.19.40
+                entity.refreshBoundingBox();
+                entity.updateBedrockMetadata();
```

Comment:
You can probably just call this in `refreshBoundingBox()` ?

Reply:
Thanks, didn't realize how often the entity data was sent